### DISCUSSION TITLE
After adding a domain the API returns verified=true

### DISCUSSION THREAD

**Discussion by mikigraf** [vercel/community#3619] [Help] [answered]
### Summary

Hello,

I'm in the midst of implementing a feature that allows users to attach their custom domains to our application. When we interact with the projects/{teamID}/domains endpoint, we don't receive a verification challenge. Subsequently, when we invoke the domain verification endpoint, it responds with the following:
`{
    "verified": true,
    "domainJson": {
        "name": "x",
        "apexName": "x",
        "projectId": "123",
        "redirect": null,
        "redirectStatusCode": null,
        "gitBranch": null,
        "updatedAt": 1692096197405,
        "createdAt": 1692096197405,
        "verified": true
    }
}`

It's only after we manually configure the A records for Vercel that we get the TXT record (with verified=false) to authenticate the domain.

I've noticed that on vercel.com, users can directly obtain the TXT record post domain addition. Is there a way to mirror this behavior when using the API, considering it seems to function differently?

### Steps to Reproduce

Add a domain to a project via API

**Comment by smaeda-ks** [comment, marked as answer]
@mikigraf I think you need to clarify your exact testing scenario. 

https://vercel.com/docs/rest-api/endpoints#get-a-project-domain

The endpoint above returns `verification` field regardless of the domain's DNS lookup status if it's not verified yet. Make sure to pass the `teamId` query parameter when you test this behavior using multiple teams.
